### PR TITLE
Renamed the package to conform to the group standards.

### PR DESCRIPTION
**Pull request by alex-ko-dev**
The base package was renamed from com.cx.configprovider to com.checkmarx.configprovider to be consistent with CxFlow, CxSDK and the plugins.

### CODE REVIEW

File: src/main/java/com/checkmarx/configprovider/ConfigProvider.java [hunk 1]
```diff
@@ -1,9 +1,9 @@
-package com.cx.configprovider;
+package com.checkmarx.configprovider;
 
-import com.cx.configprovider.dto.interfaces.ConfigResource;
+import com.checkmarx.configprovider.dto.interfaces.ConfigResource;
```
Comment (ofersk) [inline src/main/java/com/checkmarx/configprovider/ConfigProvider.java:3]: Package names are written in all lower case

**Review by ofersk — CHANGES_REQUESTED** [review]
Package names are written in all lower case, there are a lot of packages here that fail this.
this could lead  to problems and is against java's specs

File: src/main/java/com/checkmarx/configprovider/ConfigProvider.java (hunk 1 above)
Comment (alex-ko-dev) [reply src/main/java/com/checkmarx/configprovider/ConfigProvider.java:3]: Isn't it lower case already?

**Review by ofersk — APPROVED** [review]

**Review by ofersk — APPROVED** [review]

**Review by ofersk — APPROVED** [review]

File: src/main/java/com/checkmarx/configprovider/ConfigProvider.java (hunk 1 above)
Comment (ofersk) [reply src/main/java/com/checkmarx/configprovider/ConfigProvider.java:3]: my bad

**Review by ofersk — APPROVED** [review]

**Review by NimrodGolan — APPROVED** [review]

**Pull request merged**